db/queries/queries_api_views.sql

```diff
@@ -0,0 +1,25 @@
+-------------------------------------------------------------------
```

Comment:
I don't see the need to prefix the file names with `queries`. They're already in the queries folder

Reply:
It’s more so I can find all the queries by using *queries_* in my IDE and to make it clear that although these are in the queries folder these are the ones that were generated. Otherwise it’s not immediately apparent which files are internal to sqlc and which are the queries